File: fib_let.c

int fib(n) {
	if (n == 0) {
		return 1;
	} else if (n == 1) {
		return 1;
	} else {
		int r1 = fib(n-1);
		int r2 = fib(n-2);
		return r1 + r2;
	}
}
int main(int argc, char **argv) {
	printf("%d\n", fib(atoi(argv[1])));
	return 0;
}

File: fib_let.py
import sys
def fib(n):
    if n == 0:
        return 1
    elif n == 1:
        return 1
    else:
        r1 = fib(n-1)
        r2 = fib(n-2)
        return r1 + r2
print(fib(int(sys.argv[1])))
